### ISSUE TITLE
`Cannot read property 'suggestedNames' of undefined` when re-exporting a glob imports

### ISSUE THREAD

**Issue by Swatinem** [rollup/rollup#74]
I have code roughly like this:

```js
// widgets.js
export {default as Box} from "./box"
// lib.js
import * as Widgets form "./widgets"
export {Widgets}
```

That fails with
```
TypeError: Cannot read property 'suggestedNames' of undefined
    at Module.getCanonicalName (.gobble-build/01-rollup/1/rollup.js:1234:28)
    at .gobble-build/01-rollup/1/rollup.js:2055:76
```

I haven’t been able to create a minified testcase yet.

**Comment by Rich-Harris** [comment]
Thanks for raising this @Swatinem – will be fixed by #78

**Comment by Rich-Harris** [comment]
Just released 0.14.0, so this should be fixed now. Let us know if not
